app/config/collections.php

```diff
@@ -2149,40 +2250,29 @@
                 'filters' => [],
             ],
             [
-                '$id' => ID::custom('schedule'),
+                '$id' => ID::custom('scheduleId'),
                 'type' => Database::VAR_STRING,
                 'format' => '',
-                'size' => 128,
+                'size' => Database::LENGTH_KEY,
                 'signed' => true,
                 'required' => false,
                 'default' => null,
                 'array' => false,
                 'filters' => [],
             ],
             [
-                '$id' => ID::custom('scheduleUpdatedAt'), // Used to fix duplicate executions bug. Can be removed once new queue library is used
-                'type' => Database::VAR_DATETIME,
-                'format' => '',
-                'size' => 0,
-                'signed' => false,
-                'required' => false,
-                'default' => null,
-                'array' => false,
-                'filters' => ['datetime'],
-            ],
-            [
-                '$id' => ID::custom('schedulePrevious'),
-                'type' => Database::VAR_DATETIME,
+                '$id' => ID::custom('schedule'),
+                'type' => Database::VAR_STRING,
                 'format' => '',
-                'size' => 0,
-                'signed' => false,
+                'size' => 128,
+                'signed' => true,
                 'required' => false,
                 'default' => null,
                 'array' => false,
-                'filters' => ['datetime'],
+                'filters' => [],
             ],
             [
-                '$id' => ID::custom('scheduleNext'),
+                '$id' => ID::custom('scheduleUpdatedAt'), // Used to fix duplicate executions bug. Can be removed once new queue library is used
```

Comment:
Can you reformat this section ? 
Seems like we only added one new attribute `scheduleId` but the diff looks very complex and makes it hard to review.